service/routes.py

```diff
@@ -89,6 +90,126 @@ def create_wishlists():
         status.HTTP_201_CREATED
     )
 
+######################################################################
+# RETRIEVE A WISHLIST
+######################################################################
+@app.route("/wishlists/<int:wishlist_id>", methods=["GET"])
+def get_wishlist(wishlist_id):
+    """
+    Retrieve a single wishlist
+    This endpoint will return a wishlist based on it"s id
+    """
+    app.logger.info("Request for wishlist with id: %s", wishlist_id)
+    wishlist = Wishlist.find_by_id(wishlist_id)
+    if not wishlist:
+        return(
+            f"Wishlist with id {wishlist_id} was not found",
+            status.HTTP_404_NOT_FOUND
+        )
+    return make_response(jsonify(wishlist.serialize()), status.HTTP_200_OK)
+
+
+######################################################################
+# ADD A PRODUCT IN A WISHLIST
+######################################################################
+@app.route("/wishlists/<int:wishlist_id>/products", methods=["POST"])
+def add_product_to_wishlist(wishlist_id):
+    """
+    This endpoint will add a product into wishlist
+    """
+    app.logger.info("Request to add a product")
+    check_content_type("application/json")
+
+    data = request.get_json()
+    product_id = data["id"]
+    product_exist_check = Product.find_by_id( product_id )
+    if not product_exist_check:
+        return(
+            f"Product with id {product_id} was not found",
+            status.HTTP_404_NOT_FOUND
+        )
+    
+    wishlistproduct = WishlistProduct()
+    wishlistproduct.product_id = product_id
+    wishlistproduct.wishlist_id = wishlist_id
+    if not Wishlist.find_by_id( wishlist_id ):
+        return(
+            f"Wishlist with id {wishlist_id} was not found",
+            status.HTTP_404_NOT_FOUND
+        )
+    wishlistproduct.create()
+    message = wishlistproduct.serialize()
+
+    return make_response(jsonify(message), status.HTTP_201_CREATED)
```

Comment:
Would this be a `201`? You're technically not creating a new resource, right? I think just a `200` is better.